test/smoketest/smoketest-update.bash

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+# Put these somewhere else before running via travis
+export TRAVIS_BUILD_DIR=`git rev-parse --show-toplevel`
```

Comment:
Is this meant to be run by Travis? If so, these variables should just be in the script's environment. I don't _think_ you should have to re-export them for the child scripts, either. Did that turn out to be necessary?